bonusopdracht.js

```diff
@@ -0,0 +1,49 @@
+// Bonusopdracht
+// Maak drie knoppen op de pagina: Sorteer op prijs, AmbiLight TV's en Uitverkochte exemplaren.
+// Gebruik de code die je in opdracht 1b, 1c en 1d hebt gemaakt en schrijf dit om naar functies zodat je ze kunt aanroepen
+// op het moment dat de buttons geklikt worden.
+// Zorg ervoor dat de functies de uitkomsten in de console loggen als de gebruiker op de bijbehorende knop klikt.
+// Tip: lees hiervoor paragraaf 7.4 op EdHub eens door!
+// Zorg er nu voor, in plaats van dat de uitkomsten in de console worden gelogd,
+// dat de uitkomsten worden meegegeven aan jouw generateTV-functie
+// zodat de resultaten daadwerkelijk op de pagina weergegeven worden!
+
+function sortPrice () {
+    const sorted = inventory.sort((a, b) => {
+        if (a.price < b.price) {
+            return -1;
+        }
+        else if (a.price > b.price) {
+            return 1;
+        }
+        else return 0;
+    })
```

Comment:
Goed hoor! Maar het hoeft niet perse `-1` of `1` te zijn. Het gaat erom dat het een positief of negatief getal is. Je zou het zo kort kunnen schrijven als:
```suggestion
    const sorted = inventory.sort((a, b) => a.price - b.price);
```